README.md

```diff
@@ -41,4 +41,4 @@ The `rename-project.bash` script takes a single argument - the name of the proje
 * The project description (class name with spaces between the words)
 * The main package name (project name with hyphens removed)
 
-It then performs a search and replace and directory renames so the project is ready to be used.
\ No newline at end of file
+It then performs a search and replace and directory renames so the project is ready to be used.
```

Comment:
Not part of this ticket. Just an FYI - have created a Tech Debt ticket https://dsdmoj.atlassian.net/browse/VB-4856 to update README